packages/ResponsesAccordion/src/components/Indicator/Indicator.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+import CheckIcon from "@paprika/icon/lib/Check";
+import { indicatorStyles, indicatorDotStyles } from "./Indicator.styles";
+
+const propTypes = {
+  isActive: PropTypes.bool,
+  isComplete: PropTypes.bool,
+  isLast: PropTypes.bool,
+};
+
+const defaultProps = {
+  isActive: false,
+  isComplete: false,
+  isLast: false,
+};
+
+const Indicator = props => {
+  const { isComplete, isActive } = props;
+
+  return (
+    <div data-pka-anchor="indicator" css={indicatorStyles} {...props}>
+      <span data-pka-anchor="indicator.dot" css={indicatorDotStyles} {...props}>
```

Comment:
Seems unlikely we really need to spread all props onto two different elements

Reply:
I need 3 of 3 for `indicatorStyles` and 2 of 3 for `indicatorDotStyles`.  Seems simplest to just pass them all through, though maybe it would read better to declare explicit `xStylingProps` for each of them...?